cantabular/static_dataset_test.go

```diff
@@ -422,6 +422,42 @@ func TestGetDimensionsByNameHappy(t *testing.T) {
 	})
 }
 
+func TestGetGeographyDimensionsHappy(t *testing.T) {
+	Convey("Given a correct getGeographyDimensions response from the /graphql endpoint", t, func() {
+		testCtx := context.Background()
+
+		mockHttpClient := &dphttp.ClienterMock{
+			PostFunc: func(ctx context.Context, url string, contentType string, body io.Reader) (*http.Response, error) {
+				return Response(
+					[]byte(mockRespBodyGetGeographyDimensions),
+					http.StatusOK,
+				), nil
+			},
+		}
+
+		cantabularClient := cantabular.NewClient(
+			cantabular.Config{
+				Host:       "cantabular.host",
+				ExtApiHost: "cantabular.ext.host",
+			},
+			mockHttpClient,
+			nil,
+		)
+
```

Comment:
Missing a test to verify that the query is correct

Reply:
Yes, in fact, this was missing for all the tests. I added a few aux funcs to help create mocks and validate queries.